llvm/test/CodeGen/AIE/aie2/live-reserved-regs-call.ll

```diff
@@ -0,0 +1,55 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc -mtriple=aie2 -O2 --issue-limit=1 -verify-machineinstrs %s -o - 2>&1 | FileCheck %s
+
```

Comment:
please add a comment describing what we test here, after months just by file name it becomes tricky to remember the use/need of the test :)

Reply:
done.